ml/src/main/scala/ai/h2o/sparkling/ml/params/HasMonotoneConstraints.scala

```diff
@@ -0,0 +1,49 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package ai.h2o.sparkling.ml.params
+
+import java.util
+
+import hex.KeyValue
```

Comment:
Do we wan to expose H2O structure in this API?

Reply:
`KeyValue` is used only by (this method)[https://github.com/h2oai/sparkling-water/pull/1678/files/beea788bee0125e586d15eb3e2e0c13a903a2b45#diff-dcdd1785c6fc00ccd365f4e5dce628afR39], which serves only for internal usage. The result of this method is immediately passed to H2O-3 internals. E.g.: [``XGBoost``](https://github.com/h2oai/sparkling-water/pull/1678/files/beea788bee0125e586d15eb3e2e0c13a903a2b45#diff-e98c0636636d3757ad23131bcad2b095R369)

Reply:
👍